apps/web/components/sections/project/update/update-card.tsx

```
@@ -14,115 +1,227 @@
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+} from "~/components/base/alert-dialog";
+import { UpdateForm } from "./update-form";
+import { formatDistanceToNow } from "date-fns";
 
 interface UpdateCardProps {
-	data: UpdateItem[]
-	updatesUrl: string
+  data: any[];
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)